public/app/core/internationalization/constants.ts

```diff
@@ -24,6 +40,112 @@ export interface LanguageDefinition<Namespace extends string = string> {
   loader: Record<Namespace, LocaleFileLoader>;
 }
 
+const EXTRA_LANGUAGES: LanguageDefinition[] = [
+  {
+    code: CHINESE_TRADITIONAL,
+    name: '中文（繁體）',
```

Comment:
space intentional here?

Reply:
Nice catch! (furiously googles)

There's not actually a space there - it's a `U+FF09 : FULLWIDTH RIGHT PARENTHESIS` (compared to a 'normal' `) U+0029 : RIGHT PARENTHESIS`. It's apparently standard in Chinese so they take up a full character-width https://en.wikipedia.org/wiki/Chinese_punctuation#Marks_similar_to_European_punctuation 

It's the same for the existing CHINESE_SIMPLIFIED

Reply:
TIL 🤔